app/src/main/java/com/example/moviedb/ui/popular/PopularFragment.kt

```diff
@@ -20,6 +21,8 @@ class PopularFragment : BaseFragment<FragmentPopularBinding>() {
         popularViewModel = ViewModelProviders.of(this).get(PopularViewModel::class.java)
         popularViewModel.movieResponseLiveData.observe(this, Observer {
             val adapter = MovieAdapter(it.movies)
```

Comment:
Đoạn này sao cần phải dùng lại biến phụ làm gì đâu nhỉ
```suggestion
            mBinding.recyclerMovies.apply {
                layoutManager = GridLayoutManager(activity, COLUMN_NUMBER)
                adapter = MovieAdapter(it.movies)
            }
```